test-apps/display-test-app/src/frontend/App.ts

```diff
@@ -176,6 +176,13 @@ export class DisplayTestApp {
       localhostIpcApp: {
         socketPort: 3002,
       },
+      nativeApp: {
+        authConfig: {
+          clientId: "imodeljs-electron-test",
+          redirectUri: "imodeljs://app/signin-callback",
+          scope: "openid email profile organization imodelhub context-registry-service:read-only reality-data:read product-settings-service projectwise-share urlps-third-party imodel-extension-service-api imodeljs-router offline_access",
+        },
+      }

```

Comment:
all changes to this file should be removed.

Reply:
this should be done on the backend.

Reply:
it works